src/ocamlorg_frontend/pages/playground.eml

```diff
@@ -117,10 +128,16 @@ let render ~default_code =
                 class="h-auto flex-col overflow-auto p-4 text-white"
                 x-show="output && !mobile"
                 ></div>
+               
             </div>
           </div>
         </div>
       </div>
     </div>
+     <script>
+      
+            document.getElementById("clear-output").addEventListener("click", function() {
+            document.getElementById("output").innerHTML = ""  })
+          </script>
```

Comment:
please tidy up indentation/whitespace

Reply:
> please tidy up indentation/whitespace

Good morning Sabine, I am done with this. Kindly review